### PR TITLE
PRO-7583: updating privacy policy addresses and email

### PR DESCRIPTION
**Pull request by elvianihuseyin**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: app/utils/FeesLookup.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const createCall = async (applicationId, data, authToken) => {
         status: 'success',
         total: 0
     };
-    logInfo(`Sending fee request to api with the following payload: ${JSON.stringify(data)}`, applicationId);
+    logInfo(`Sending fee request to api with the following payload: ${JSON.stringify(data)} for auth=` + authToken, applicationId);
```
Comment (sanjayparekh-hmcts) [inline app/utils/FeesLookup.js:32]: can you revert this change please - dont know how it got in

Comment (LucaDelBuonoHMCTS) [reply app/utils/FeesLookup.js:32]: Looks like you committed it yourself :)

Comment (LucaDelBuonoHMCTS) [reply app/utils/FeesLookup.js:32]: See the commit at the top: https://github.com/hmcts/probate-caveats-frontend/pull/285/commits

Comment (sanjayparekh-hmcts) [reply app/utils/FeesLookup.js:32]: yes I know but I cant remove it

Comment (sanjayparekh-hmcts) [reply app/utils/FeesLookup.js:32]: whats all the autocomplete off changes?

Comment (LucaDelBuonoHMCTS) [reply app/utils/FeesLookup.js:32]: Ok, I'll remove it for you

Comment (LucaDelBuonoHMCTS) [reply app/utils/FeesLookup.js:32]: So the a11y tests don't fail.

Comment (LucaDelBuonoHMCTS) [reply app/utils/FeesLookup.js]: That change on FeesLookup.js is now gone.
